app/assets/javascripts/deploys.js

```diff
@@ -176,6 +176,13 @@ $(function () {
     }
   }
 
+  $("#recent-releases .release-label").on('click', function(event){
+    event.preventDefault();
+    // Get version number from link href
+    var version = event.target.href.substring(event.target.href.lastIndexOf('/') + 1);
```

Comment:
I dunno, I prefer this way because there's no need to actually split it into all the parts.